modules/manage/partials/monitor-health.adoc

```diff
@@ -4,6 +4,16 @@ This section provides guidelines and example queries using Redpanda's public met
 
 TIP: To help detect and mitigate anomalous system behaviors, capture baseline metrics of your healthy system at different stages (at start-up, under high load, in steady state) so you can set thresholds and alerts according to those baselines.
 
+[TIP]
+====
+For counter type metrics, a broker restart causes the count in tools such as Prometheus and Grafana to reset to zero. Redpanda recommends wrapping counter metrics in a rate query to account for broker restarts, for example:
+
+[,promql]
+----
+rate(redpanda_kafka_records_produced_total)
```

Comment:
@wzzzrd86 is this how users would use the rate function with this metric?

Reply:
I think rate needs a time frame as well, but I could be wrong, something lieke rate(redpanda_kafka_records_produced_total [5m])